ros_cross_compile/docker/gather_rosdeps.sh

```diff
@@ -32,6 +32,7 @@ rosdep install \
     --ignore-src \
     --reinstall \
     --default-yes \
+    --skip-keys "${SKIP_ROSDEP_KEYS}" \
```

Comment:
What happens when SKIP_ROSDEP_KEYS is empty? does this work properly?

Reply:
yes this works properly